Pear/Models/User.swift

```diff
@@ -10,29 +10,19 @@ import Foundation
 
 struct User: Codable, Equatable {
 
+    let netID: String
     let firstName: String
-    let goals: [String]
-    let googleID: String
-    let graduationYear: String?
-    let groups: [String]
-    let hometown: String
-    let interests: [String]
     let lastName: String
-    let major: String
-    let netID: String
+    let hometown: String
     let profilePictureURL: String
+    let major: String
+    let graduationYear: String?
     let pronouns: String
-    let facebook: String?
-    let instagram: String?
-    let talkingPoints: [String]
-    let availabilities: [DaySchedule]
-    let preferredLocations: [Location]
-    let matches: [Match]
+    let interests: [String]
+    let groups: [String]
 
     static func == (lhs: User, rhs: User) -> Bool {
         lhs.firstName == rhs.firstName &&
-        lhs.goals == rhs.goals &&
-        lhs.googleID == rhs.googleID &&
         lhs.graduationYear == rhs.graduationYear &&
```

Comment:
Not sure if you enforce any invariants on the backend, but this could probably just be `lhs.netID == rhs.netID` since a user should not be able to sign up twice w/ the same ID (something would be broken then). Just a thought, but feels like cheating.